backend/controllers/servicenamespace/servicenamespace_reconcile.go

```diff
@@ -314,3 +322,74 @@ func (c *reconciler) listAPIServiceExports(ctx context.Context, cache cache.Cach
 	}
 	return exports, nil
 }
+
+func (c *reconciler) cleanupRBACResources(
+	ctx context.Context,
+	client client.Client,
+	cache cache.Cache,
+	sns *kubebindv1alpha2.APIServiceNamespace,
+	apiServiceExports *kubebindv1alpha2.APIServiceExportList,
+) error {
+	expected := make(map[string]bool)
+	for _, export := range apiServiceExports.Items {
+		name := fmt.Sprintf("kube-binder-export-%s-%s", sns.Name, export.Name)
+		expected[name] = true
+	}
+	prefix := fmt.Sprintf("kube-binder-export-%s-", sns.Name)
```

Comment:
This variable can be moved up and then also be used inside the loop to construct the object names. Reducing code duplication.